src/shapes/Line.ts

```diff
@@ -51,7 +51,7 @@ function expandPoints(p, tension) {
 }
 
 export interface LineConfig extends ShapeConfig {
-  points?: number[];
+  points?: [number, number, number, number, number, number];
```

Comment:
Guess this one is incorrect. 
A line can have dynamic amount of points

Reply:
Good catch. I got deceived by the example in the docs explicitly listing 3 pairs of points.